lighthouse-core/audits/predictive-perf.js

```diff
@@ -98,7 +122,22 @@ class PredictivePerf extends Audit {
       let sum = 0;
       const values = {};
       Object.keys(graphs).forEach(key => {
-        values[key] = PageDependencyGraph.computeGraphDuration(graphs[key]);
+        const estimate = PageDependencyGraph.estimateGraph(graphs[key]);
+        const lastLongTaskEnd = PredictivePerf.getLastLongTaskEndTime(estimate.nodeTiming);
+
+        switch (key) {
```

Comment:
nit: I would personally prefer `if`/`else if` here since it's not really a loop (though really the `Object.keys` may not be needed at all since it's four fixed, known cases that have to be enumerated above and below anyways :P)

Reply:
yeah but those duplicated 3 lines at the top really look pretty bad 😆